plugin/trino-opensearch/src/test/java/io/trino/plugin/opensearch/TestOpensearch2ConnectorTest.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.opensearch;
+
+import com.google.common.collect.ImmutableList;
+import org.junit.jupiter.api.parallel.Isolated;
+
+import java.util.List;
+
+@Isolated
+public class TestOpensearch2ConnectorTest
+        extends BaseOpensearchConnectorTest
+{
+    public TestOpensearch2ConnectorTest()
+    {
+        super("opensearchproject/opensearch:2.2.0", "opensearch");
```

Comment:
Why not the latest - 2.11.0?

Reply:
Client is latest, server is deliberately older :)